src/constants.ts

```diff
@@ -75,3 +75,21 @@ export const questionTakenFileName = {
   message:
     "Looks like there's already a file with that name. What would you like to name your file?",
 };
+
+// ------------------------
+// ----- DESCRIPTIONS -----
+// ------------------------
+export const ADD_DESCRIPTION =
+  '\nairfoil add [env | appcenter | adr | keystore]\n\nUsage:\n\nairfoil add env # if args omitted -> prompts to get ENV key, value\nairfoil add env MY_VAR=mydatahere\nairfoil add appcenter # follow prompts\nairfoil add adr # follow prompts\nairfoil add adr "Choose Cognito for Auth" # specify ADR title as arg1\nairfoil add keystore\n\nFor more information, visit the official documentation:\nhttps://airfoil-docs.herokuapp.com/commands/add';
+
+export const CONVERT_DESCRIPTION =
+  '\nairfoil convert [svg]\n\nUsage:\n\nairfoil convert svg\nairfoil convert svg --cleanup # to remove app/assets/svg after successful conversion\n\nConverts .svg files to .tsx files for use in React Native. For more information, visit the official documentation:\nhttps://airfoil-docs.herokuapp.com/commands/convert';
+
+export const CREATE_DESCRIPTION =
+  '\nairfoil [create | c]\n\nUsage:\n\nairfoil create # guides you through a quick wizard\nairfoil create component [ComponentName] [-f folderName]\nairfoil create hook [useHookName] [-f folderName]\n\nCreates a file and places it in the correct directory. Currently supported options:\n- component\n- hook. For more information, visit the official documentation:\nhttps://airfoil-docs.herokuapp.com/commands/create';
+
+export const INIT_DESCRIPTION =
+  '\nairfoil [init | i | new] [nameOfProject]\n\nUsage:\n\nairfoil init myAwesomeApp\nairfoil init # prompts you for the <appName>\n\nInitializes a new React Native project. For more information, visit the official documentation:\nhttps://airfoil-docs.herokuapp.com/commands/init';
+
+export const VERSION_DESCRIPTION =
```

Comment:
This is more of a coding preference, but could we change the var names to the following:

```
HELP_DESCRIPTION_CMD_ADD
HELP_DESCRIPTION_CMD_CONVERT
HELP_DESCRIPTION_CMD_CREATE
// etc.
```

I personally like this manner of var names as when searching in Intellisense, you'll get autofill like the following:

Start typing: "HELP_DESC..."
All of the above show up as possibilities.

Additionally, this acts as a pseudo name-spacing strategy. Never have to worry about name clashes because this is so specific.

What do you think?

Reply:
I'm down for that. I'll update that real quick! @townofdon